src/Certify.Tests/Certify.Core.Tests.Unit/CertifyManagerTests.cs

```diff
@@ -0,0 +1,174 @@
+﻿using System;
+using System.Threading.Tasks;
+using Certify.Config.Migration;
+using Certify.Management;
+using Certify.Models;
+using Certify.Service;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Certify.Core.Tests.Unit
+{
+    [TestClass]
+    public class CertifyManagerTests
+    {
+        private readonly CertifyManager _certifyManager;
+
+        public CertifyManagerTests()
+        {
+            _certifyManager = new CertifyManager();
+            _certifyManager.Init().Wait();
+        }
+
+        [TestMethod, Description("Happy path test for using CertifyManager.GetACMEProvider()")]
+        public async Task TestCertifyManagerGetACMEProvider()
+        {
+            // Setup test data
+            var testUrl = "test.com";
+            var dummyManagedCert = (new ManagedCertificate { CurrentOrderUri = testUrl, UseStagingMode = true });
+            var caAccount = await _certifyManager.GetAccountDetails(dummyManagedCert);
+
+            // Get results from CertifyManager.GetACMEProvider()
+            var acmeClientProvider = await _certifyManager.GetACMEProvider(dummyManagedCert, caAccount);
+
+            // Validate return from CertifyManager.GetACMEProvider()
+            Assert.IsNotNull(acmeClientProvider, "Expected response from CertifyManager.GetACMEProvider() to not be null");
+            Assert.AreEqual("https://acme-staging-v02.api.letsencrypt.org/directory", acmeClientProvider.GetAcmeBaseURI());
```

Comment:
This is assuming that we always have an LE staging account as our default (and it's created), we should probably define the expected CA directory URL in test config somewhere.

Reply:
This has been refactored to grab value from CertificateAuthority.CoreCertificateAuthorities